Calculator/ViewController.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  ViewController.swift
+//  Calculator
+//
+//  Created by Oleksii Kolomiiets on 4/4/18.
+//  Copyright © 2018 Oleksii Kolomiiets. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        inTheMiddleOfTyping = false
+    }
+    
+    @IBOutlet weak var display: UILabel!
+    
+    private var inTheMiddleOfTyping: Bool!
+    var result = 0.0 {
+        didSet {
+            displayValue = result
+        }
+    }
+    var previousValue = 0.0
+    var operation = ""
```

Comment:
I'd recommend make variable names at least with 2 words.. I.e. sometimes it's not clear what is the purpose of operation for controller...

Reply:
Got it. I will change the "operation"  and will be keeping to this approach.